portal/src/graphql/portal/LoginIDConfigurationScreen.tsx

```diff
@@ -450,6 +513,41 @@ const AuthenticationLoginIDSettingsContent: React.FC<AuthenticationLoginIDSettin
           "LoginIDConfigurationScreen.email.blockPlusTooltipMessage"
         )}
       />
+      <CheckboxWithContent
```

Comment:
I have a suggestion here. Instead of disabling the checkboxes, can we smartly change the checked status of the checkboxes based on what have been clicked?

For example, if allowlist is checked, then blocklist is disabled and block free provider is disabled.

I think this is more user-friendly.

Reply:
Discussed with Carmen and we decided to keep the current implementation.